### DISCUSSION TITLE
How are you dogfooding?

### DISCUSSION THREAD

**Discussion by dghaehre** [git-bug/git-bug#985] [Q&A] [answered]
Caveat: I was unable to get v8.0 working ("auth required" when pushing), so I used aur and installed 7.2 and that worked.

I then cloned this repo, ran "git bug user create", "git bug pull origin" and expecting to get to see how this work with a project with real issues. But it seems to not pull any issues. Is this because I did something wrong or are you not using git-bug for this repo?

**Comment by mcepl** [comment]
Let’s hope that the author at least uses GitHub bridge.

**Comment by sudoforge** [comment, marked as answer]
:wave: hey there! Thanks for asking this question.

Unfortunately, this repository is not (yet) dogfooding `git-bug`. I am a long time user, new maintainer, and am working to revitalize development on this project. I actually created #1221 (about dogfooding git-bug) prior to seeing this! I think it is **incredibly** important to get the project to a state that works for the contributors maintaining it and the community using it.

I've been using `git-bug` personally for several years. To my mind, there are a different tracks that could be considered important for what I'd consider a `1.0`-ish release, or _something that can fully replace existing project management platforms_.  At a high level, these are:

- Identity security (preventing impersonation, say, with required commit signatures)
- Sprints and Grouped Views (think something like GitHub Milestones or Projects, or a Jira Board, where you have some collection of issues. this could be entirely based on structured, well-known/reserved label keys, such as `project` or `sprint`)
- Issue visibility control (using encryption to manage access to issues)
- An RFC system and commitment to a stable interface. I found this project years ago as an individual, but if we want to be serious about being a viable option in the project management space (and I do!), we will need a structured community approach to review, acceptance, and rejection of change proposals.
- API improvements, particularly around authentication and authorization. We need to be able to prevent certain actions from happening, such as managing settings (when those are introduced), or projects (when those are introduced).
- Deeper bridge integration. This is a topic I've been mulling about, and largely, I think focusing on very deep integration with GitHub and GitLab are key to success. I'll be talking more about this in the future through issues and discussions.

I'm hoping to be able to catch up on the backlog of issues and PRs that exist and cut a new release soon to include commits from the past two years. After that, I'll be turning my attention to improving `git-bug` so that it can fully replace a project management workflow one might be using GitHub, GitLab, or another platform for.

If you're interested in helping out, let me know! I'll be sure to reach out when I start roadmapping these items out.

---

That said, I've been very happy using `git-bug` personally, without bridges, for many years. If you're using as a personal "light issue tracker", I think you'll find that it's very capable in that regard. If you need more full-featured project management support, well, keep an eye out for the future!
